### PR TITLE
Move jquery.contactsmenu.js to compiled handlebars

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

File: core/js/jquery.contactsmenu.js [hunk 1]
```diff
@@ -71,7 +64,7 @@
 				}
 
 				actions.forEach(function(action) {
-					var template = Handlebars.compile(ENTRY);
```
Comment (ChristophWurst) [inline core/js/jquery.contactsmenu.js]: what about the `LIST` template?

Comment (rullzer) [reply core/js/jquery.contactsmenu.js]: It is not really a template. Just a long string. As you can see it is also just used as is. Not compiled at all.

**Review by ChristophWurst — APPROVED** [review]
Looks good :+1:

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
@rullzer Yes, basically some whitespace. I adjusted the tests to remove all whitespace Seems fine now https://drone.nextcloud.com/nextcloud/server/11547/283

**Pull request merged**